resharper/resharper-unity/test/data/CSharp/Daemon/UsageChecking/PreprocessBuildInterface01.cs

```diff
@@ -0,0 +1,19 @@
+using UnityEditor;
+using UnityEditor.Build;
+
+public class A : IPreprocessBuild
+{
+    public int callbackOrder { get; }
+    public void OnPreprocessBuild(BuildTarget target, string path)
+    {
+        throw new System.NotImplementedException();
+    }
+}
+
+public class B : A, IPostprocessBuild
+{
+    public void OnPostprocessBuild(BuildTarget target, string path)
+    {
+        throw new System.NotImplementedException();
+    }
+}
```

Comment:
Add another case where the interface is being implicitly implemented via a base class, e.g.

```
public abstract class Base : IPostprocessBuild
{
  // …
  public abstract void OnPostProcessBuild();
}
```

and

```
public class Derived : Base
{
  public override void OnPostProcessBuild(…) { }
}
```

This should also help check if we can short circuit the type check.